modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -302,6 +311,10 @@ function getFeatureUniqueId(feature, uniqueIdProperty) {
   return -1;
```

Comment:
This should return `undefined`?

Reply:
Changed. Note that this will alter behavior in that `const isFeatureIdPresent = isFeatureIdDefined(hoveredFeatureId)` (line 204) will return false for features without an unique id. I think this is correct as the old behavior would return true and then highlight the wrong feature